src/wallet/wallet.cpp

```diff
@@ -3412,7 +3413,7 @@ bool CWallet::ConvertList(std::vector<CTxIn> vecTxIn, std::vector<CAmount>& vecA
 bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletTx& wtxNew, CReserveKey& reservekey, CAmount& nFeeRet,
                                 int& nChangePosInOut, std::string& strFailReason, const CCoinControl* coinControl, bool sign, AvailableCoinsType nCoinType, bool fUseInstantSend, int nExtraPayloadSize)
 {
-    CAmount nFeePay = fUseInstantSend ? CTxLockRequest().GetMinFee() : 0;
```

Comment:
Shouldn't we check for activation of auto-tx here?

Reply:
There is a condition at the end which will restart the loop if the fee is less than the required one for each case, so should be fine imo. But on the other hand, this change looks irrelevant and shouldn't be a part of this PR at all probably 🤔